CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-# dbt_iterable_source version.version
+# dbt_iterable_source v.0.10.1
+[PR #42](https://github.com/fivetran/dbt_iterable_source/pull/42) includes the following updates:
```

Comment:
We should adjust the CHANGELOG to link the PR on the individual entry lines since there are other updates in this release.

Reply:
updated